nodes/src/nodes/audio_tts/IInstance.py

```
@@ -0,0 +1,101 @@
+# SOFTWARE.
+# =============================================================================
+
+import base64
+import json
+import os
+
+from rocketlib import IInstanceBase, AVI_ACTION, warning
+from .IGlobal import IGlobal
+
+
+def _infer_output_format(want_audio: bool, want_text: bool, engine: str) -> str:
```

Ruff (0.15.7)

[warning] Boolean-typed positional argument in function definition

(FBT001)